### PR TITLE
rdma-core: update to 24.7

### PR DESCRIPTION
**Pull request by uhohspaghetios**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/rdma-core/template [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-version=23.3
+version=24.7
 revision=1
 build_style=cmake
 configure_args="-DENABLE_VALGRIND=OFF"
 hostmakedepends="pkg-config python3"
 makedepends="libnl3-devel eudev-libudev-devel python3-devel"
 short_desc="RDMA core userspace libraries and daemons"
-maintainer="Rich G <rich@richgannon.net>"
-license="GPL-2.0-or-later, BSD-2-Clause-FreeBSD"
```
Comment (paper42) [inline srcpkgs/rdma-core/template:11]: why is this change necessary?

**Comment by paper42** [comment]
* squash the commits
* add the commit email to your github profile so github knows the commit is from you

**Comment by paper42** [comment]
37.1 is the latest version, is there a reason why you updated only to 24.7?

**Comment by uhohspaghetios** [comment]
Redoing this with latest version.
